site/src/pages/DeploymentSettingsPage/GeneralSettingsPage/GeneralSettingsPage.tsx

```diff
@@ -1,6 +1,9 @@
 import { deploymentDAUs } from "api/queries/deployment";
 import { entitlements } from "api/queries/entitlements";
 import { availableExperiments, experiments } from "api/queries/experiments";
+import { insightsUserStatusCounts } from "api/queries/insights";
```

Comment:
This query is meant for the licenses page. Not the general settings page. The graph on this page already has the right data. It just needs to be migrated to shadcn.